minifi-assembly/src/main/assembly/dependencies.xml

```diff
@@ -107,7 +107,7 @@
             </includes>
             <unpack>true</unpack>
             <unpackOptions>
-                <filtered>true</filtered>
+                <filtered>false</filtered>
```

Comment:
I didn't see any values that were actually being filtered. I set it to `false` so it would not interfere with the Commons Daemon executables. If I missed some let me know...